modules/tensor_mechanics/src/kernels/lagrangian/TotalLagrangianStressDivergence.C

```diff
@@ -124,7 +123,9 @@ TotalLagrangianStressDivergence::eigenstrainJacobianComponent(unsigned int cvar,
     B = _def_grad[_qp];
   }
 
-  RankFourTensor U = 0.5 * (A.mixedProductIkJl(B.transpose()) + A.mixedProductIlJk(B.transpose()));
```

Comment:
This probably belongs in a separate PR, but I won't object.

Edit: or maybe this is from one of the others...